### PR TITLE
Clean doc

### PR DESCRIPTION
**Pull request by cor3ntin**
Doc for the aseba language as well as for the natives & tymio apis, ported from https://www.thymio.org/

### CODE REVIEW

File: doc/en/aseba-std-natives.rst [hunk 1]
```diff
@@ -0,0 +1,131 @@
+Native functions standard library
+=================================
+
+``math.copy(A,B)``
+-------------------
+
+  Copy the array :math:`B` in the array :math:`A`, element by element: :math:`A_{i} = B_{i}`.
```
Comment (stephanemagnenat) [inline doc/en/aseba-std-natives.rst]: Do we need an indentation in this file?

File: doc/en/aseba-language.rst [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+::
+
+    var a
+    var b = 0
+    var c = 2*a + b        # warning: 'a' is not initialised
+
+Reserved keywords
+~~~~~~~~~~~~~~~~~
+
+` <>`__
```
Comment (stephanemagnenat) [inline doc/en/aseba-language.rst]: What is this?

File: doc/en/aseba-language.rst [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+
+The following keywords cannot be used as valid names for variables, as
+they are already used by the Aseba language.
+
+Keywords abs call callsub do else elseif emit end for if in onevent
```
Comment (stephanemagnenat) [inline doc/en/aseba-language.rst]: Shouldn't this be either italic or teletype or emphasized in a way?

File: doc/en/aseba-language.rst [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+    call math.dot(d, a, b, 3)
+    call math.dot(d, a, c[0:2], 3)
+    call math.dot(a[0], c[0:2], 3)
+
+What to read next?
+------------------
+
+You might be interested to read:
+
+-  `Description of the native functions standard library <asebastdnative.rs
```
Comment (stephanemagnenat) [inline doc/en/aseba-language.rst]: Broken link

File: doc/en/thymio-api.rst [hunk 5]
```diff
@@ -0,0 +1,503 @@
...
+
+Thymio updates this array at a frequency of 20 Hz and generates the
+``buttons`` event after every update. Moreover, for each of these
+buttons, when it is pressed or released, a corresponding event with the
+same name is fired.
+
+Distance sensors
+----------------
+
+.. rubric:: Horizontal
```
Comment (stephanemagnenat) [inline doc/en/thymio-api.rst]: Why isn't this a sub-sub-section?

File: doc/en/thymio-api.rst [hunk 6]
```diff
@@ -0,0 +1,503 @@
...
+   varies between 0 (no light) and 1023 (maximum light)
+-  ``prox.ground.reflected`` : amount of light received when the sensor
+   emits infrared, varies between 0 (no reflected light) and 1023
+   (maximum reflected light)
+-  ``prox.ground.delta`` : difference between reflected light and
+   ambient light, linked to the distance and to the ground colour.
+
+For each array, the index 0 corresponds to the left sensor and the index
+1 to the right sensor. As with the distance sensors, Thymio updates this
+array at a frequency of 10 Hz, , and generates the (same) ``prox`` event
```
Comment (stephanemagnenat) [inline doc/en/thymio-api.rst]: double ", ,"

File: doc/en/thymio-api.rst [hunk 7]
```diff
@@ -0,0 +1,503 @@
...
+Thymio holds many LEDs spread around its body. Most of them are
+associated with sensors and can highlight their activations: by default,
+the intensity of the LED is linked to the sensor value. However, once
+LEDs are used in the code, the programmer takes over control and they no
+longer reflect the sensor values.
+
+Native functions allow the various LEDs to be controlled. For all LEDs,
+their intensity values range from 0 (off) to 32 (fully lit).
+
+.. rubric:: The LED circle on top of the robot
```
Comment (stephanemagnenat) [inline doc/en/thymio-api.rst]: Same comment as for sensors, why not sub-sub-section?

File: doc/en/aseba-std-natives.rst (hunk 1 above)
Comment (cor3ntin) [reply doc/en/aseba-std-natives.rst]: I think it looks better ?

File: doc/en/aseba-language.rst (hunk 4 above)
Comment (cor3ntin) [reply doc/en/aseba-language.rst]: Oh, yeah, I meant to either fix that - but that one of the link I considered for removal. Do you think it's needed ? Maybe we neeed to setup read the doc before deciding

File: doc/en/thymio-api.rst (hunk 6 above)
Comment (cor3ntin) [reply doc/en/thymio-api.rst]: As it should be http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup

File: doc/en/aseba-std-natives.rst (hunk 1 above)
Comment (stephanemagnenat) [reply doc/en/aseba-std-natives.rst]: Do we have a preview in readthedocs?

File: doc/en/aseba-language.rst (hunk 4 above)
Comment (stephanemagnenat) [reply doc/en/aseba-language.rst]: I agree, we should setup read the doc first.

**Comment by cor3ntin** [comment]
We use aseba:clean_doc branch instead

**Pull request closed without merging**